Main.java

```diff
@@ -9,7 +11,21 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // Creates a scanner used for input
+    Scanner input = new Scanner(System.in);
+
+    // declare and initialize a variable to store the temperature in degrees celsius
+    int C = 18;
+
```

Comment:
You are missing the input for the question. The idea of the sample is to show you what one example might look like. What you should do is create a Scanner and read in a value that would be stored into this variable.
Also, the variable C is a poor name. Remember, it should always begin with a lowercase letter and it should describe what it is storing. A better name would be tempInC. Be sure to fix this one up to work properly!